account_account_tag_code/models/account_account_tag.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class AccountAccountTag(models.Model):
+    _inherit = "account.account.tag"
+
+    code = fields.Char(required=True, translate=False)
```

Comment:
```suggestion
    code = fields.Char(required=True)
```
char fields by default are translate=False

Reply:
I wasn't sure, thanks.